src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.cs

```diff
@@ -32,8 +32,12 @@ internal enum CborReaderState
         DoublePrecisionFloat,
         SpecialValue,
         Finished,
-        FormatError,
-        EndOfData,
+
+        FormatError_IndefiniteStringWithInvalidDataItems,
```

Comment:
I've made this change so that `SkipValue()` does more accurate reporting on format errors. Not 100% comfortable with how this looks, would be interested in a second opinion.

Reply:
Underscores aren't allowed in public names.

Are all of the different error codes really important to callers? Or is it just good enough for them to know "I broke"?  Certainly exception messages might want the data, but I'm not sure about the API need.  It might be overtesting.